packages/invoices/src/InboundShipment/DetailView/GeneralTab.tsx

```diff
@@ -35,48 +43,188 @@ const Expand: FC<{ rowData: InboundShipmentItem }> = ({ rowData }) => {
+          },
+        },
+      ],
+      [
+        'itemName',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return ifTheSameElseDefault(lines, 'itemName', '');
+            } else {
+              return rowData.itemName;
+            }
+          },
+        },
+      ],
+      [
+        'batch',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return ifTheSameElseDefault(lines, 'batch', '[multiple]');
+            } else {
+              return rowData.batch;
+            }
+          },
+        },
+      ],
+      [
+        'expiryDate',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return ifTheSameElseDefault(lines, 'expiryDate', '');
+            } else {
+              return rowData.expiryDate;
+            }
+          },
+        },
+      ],
+      [
+        'locationName',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return ifTheSameElseDefault(lines, 'locationName', '');
+            } else {
+              return rowData?.locationName;
+            }
+          },
+        },
+      ],
+      [
+        'sellPricePerPack',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return ifTheSameElseDefault(lines, 'sellPricePerPack', '');
+            } else {
+              return rowData.sellPricePerPack;
+            }
+          },
+        },
+      ],
+      [
+        'packSize',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return ifTheSameElseDefault(lines, 'packSize', '');
+            } else {
+              return rowData.packSize;
+            }
+          },
+        },
+      ],
+      [
+        'unitQuantity',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return lines.reduce(getUnitQuantity, 0);
+            } else {
+              return rowData.packSize * rowData.numberOfPacks;
+            }
+          },
+        },
+      ],
+      [
+        'numberOfPacks',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return lines.reduce(getSumOfKeyReducer('numberOfPacks'), 0);
+            } else {
+              return rowData.numberOfPacks;
+            }
+          },
+        },
+      ],
+      getRowExpandColumn(),
       GenericColumnKey.Selection,
     ],
-    { onChangeSortBy: onSort, sortBy },
-    [sortBy]
+    {},
+    []
```

Comment:
looking at this now maybe I should make a `columns` file..

Reply:
yeah, thought the same